test/analytics/tests/analytics_from_the_browser.ts

```diff
@@ -36,13 +40,12 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       );
     };
 
-    beforeEach(async () => {
+    before(async () => {
       await common.navigateToApp('home');
     });
 
-    // this test should run first because it depends on optInConfig being undefined
-    it('should have internally enqueued the "lifecycle" events but not handed over to the shipper yet', async () => {
-      const telemetryCounters = await getTelemetryCounters(2);
+    it('should see both events enqueued and sent to the shipper', async () => {
```

Comment:
Since `telemetry` is enabled, it sets `optIn: true`, so we no longer can control it from these tests. However, we still get similar effects because the shipper is registered before `optIn: true` (`telemetry` plugin does so in the `start` method)